.github/actions/e2e/install-prometheus/action.yaml

```diff
@@ -10,6 +10,9 @@ inputs:
   region:
     description: "Region to access AWS"
     required: true
+  prometheus_region:
+    description: "Prometheus region"
+    required: true
```

Comment:
Why do we need to host the prometheus in a different region from the region that we're running the cluster?

Reply:
Prometheus region is being fetched from environment variable and is different from the region where we are running the tests.